roles/transformers/tasks/dependencies-rpm.yml

```diff
@@ -10,15 +10,16 @@
         gpgcheck: yes
         gpgkey: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-{{ ansible_distribution_major_version }}
 
-    - name: Install imagemagick
-      yum:
-        name: "{{ download_location }}/{{ item | replace('imagemagick-distribution', 'imagemagick-distribution-' + imagemagick_version) }}.{{ pkg_format }}"
-        state: present
-        disable_gpg_check: true
-      loop: "{{  imagemagick_packages }}"
-
     - name: Install LibreOffice RPMs
       shell:
         cmd: "yum localinstall -y {{ download_location }}/LibreOffice_{{ dependencies_version.libreoffice }}_Linux_{{ ansible_architecture | replace('_','-') }}_rpm/RPMS/*.rpm"
         creates: /opt/libreoffice{{ dependencies_version.libreoffice | regex_search('^[0-9]+.[0-9]+') }}/program/soffice.bin
         warn: false
+
+    - name: Include RockyLinux specific tasks for ImageMagick
+      include_tasks: "imagemagick-rocky.yml"
+      when: ansible_distribution == 'Rocky'
+
+    - name: Include remaining RPM distributions tasks for ImageMagick
+      include_tasks: "imagemagick-rpm.yml"
+      when: ansible_distribution != 'Rocky'
```

Comment:
I don't like that we have splited logic here. If the reason is just the different package names or repos we should instead use per distribution variables